offerzen/global/employers/employers@6.1A.js

```diff
@@ -0,0 +1,295 @@
+
+    function getRoleTypes(formProperties) {
+      let fields = [
+        'role_type[In-office]',
+        'role_type[Fully-remote]',
+        'role_type[Hybrid]',
+      ];
+      let roleTypes = [];
+      originalLeadForm.find('.js-val-msg-roles').hide();
+
+      for (let key in formProperties) {
+        if (fields.includes(key) && formProperties[key] === 'on') {
+          if (key === 'role_type[Fully-remote]') {
+            roleTypes.push(
+              key.replace('role_type[', '').replace(']', '').replace('-', ' ')
+            );
+          } else {
+            roleTypes.push(key.replace('role_type[', '').replace(']', ''));
+          }
+          delete formProperties[key];
+        }
+      }
+      if (roleTypes.length < 1) {
+        originalLeadForm.find('.js-val-msg-roles').show();
+      }
+
+      return roleTypes;
+    }
+
+    function trackSubmission() {
+      var event = originalLeadForm.find('.js-analytics-event').text();
+      var action = originalLeadForm.find('.js-analytics-action').text();
+      var label = originalLeadForm.find('.js-analytics-label').text();
+      var category = originalLeadForm.find('.js-analytics-category').text();
+      var source = originalLeadForm.find('.js-analytics-source').text();
+
+      dataLayer.push({
+        event: event || 'Company Lead Form Submitted',
+        action: action || 'Lead Form Submitted',
+        label: label || 'Company Sign Up / Employer Landing Page',
+        category: category || 'Core',
+        source: source || 'Demand Sign Up',
+      });
+    }
+
+    function onSubmitOriginalCompanyLeadForm(token, e) {
+      originalLeadFormButton.attr('disabled', true);
+      let initialButtonValue = originalLeadFormButton.attr('value');
+      let dataWait = originalLeadFormButton.attr('data-wait');
+      originalLeadFormButton.attr('value', dataWait);
+      // get the value of the report_source query parameter should it be present and forward it onto form lead submission for analytics
+      let searchParams = new URLSearchParams(window.location.search);
+      const formData = new FormData(originalLeadForm[0]);
+      const formProperties = Object.fromEntries(formData.entries());
+      const roleTypes = getRoleTypes(formProperties);
+
+      if (originalLeadForm.parsley().validate()) {
+        originalLeadForm.find('.js-missing-fields').hide();
+        trackSubmission();
+        $.ajax({
+          type: 'POST',
+          url: '/company/form_leads',
+          data: JSON.stringify(
+            Object.assign({}, formProperties, {
+              workplace_policy: roleTypes.join(','),
+              referrer: location.href,
+              report_source: searchParams.get('report_source'),
+              'g-recaptcha-response-data': {
+                webflow: token,
+              },
+              page_variant_meta: `${window.pageVariantMeasureStart}-optimize-meta-${pageVariantMeasureEnd}`
```

Comment:
I think we need to add this to our end-to-end test payloads so tests keep running